### PR TITLE
Added issue events in comment view. Fix #123

### LINKED ISSUE #123: show closed and reopen markers inline with comments
just as it is on Github.

### PR DESCRIPTION
**Pull request by Meisolsson**
Are colors alright?
More events needed?

Closed and reopened.
screenshot_2015-02-07-22-56-09

Merged and closed.
screenshot_2015-02-08-00-51-55

### CODE REVIEW

File: app/src/main/java/com/github/mobile/ui/comment/CommentListAdapter.java [hunk 1]
```diff
@@ -110,7 +123,44 @@ public CommentListAdapter(LayoutInflater inflater, Comment[] elements,
...
+        String message = String.format("<b>%s</b> %s", event.getActor().getLogin(), event.getEvent());
+        avatars.bind(imageView(2), event.getActor());
+
+        String eventString = event.getEvent();
+
+        switch (eventString) {
+        case "closed":
+            message += " this ";
+            setText(0, ICON_ISSUE_CLOSE);
+            textView(0).setTextColor(Color.rgb(189,44,0));
```
Comment (fadils) [inline app/src/main/java/com/github/mobile/ui/comment/CommentListAdapter.java]: I think, it would be better to define all color values in xml instead.

**Comment by fadils** [comment]
Good stuff!

**Comment by Meisolsson** [comment]
Will fix the colors tomorrow, I will also later add a new issue for updating the icons so I won't add these to the TypefaceUtils file yet.

**Comment by jonanp** [comment]
Awesome work @Meisolsson!
I've wanted to have this for a long time, I'll try to review the patch today

**Comment by Meisolsson** [comment]
Ok, just wait until I fixed the string issue

**Comment by fadils** [comment]
@jonanp I think this should be merged to master ASAP. What do you think?

**Comment by jonanp** [comment]
@fadils agreed, this should go into our next release.

I expect to have the master branch ready tonight and I'd also like to upload a new version of ForkHub (has around 200 users already) to get a bit of testing before the release.

**Comment by jonanp** [comment]
This PR breaks showing a newly added comment.

I've fixed it [here](https://github.com/jonanp/ForkHub/commit/797cd237b026c39abc187872b108a269beee0f16) and added a few improvements [1](https://github.com/jonanp/ForkHub/commit/b73ae697d86f6667a9940ff5dd4737a5f2322ab4), [2](https://github.com/jonanp/ForkHub/commit/9f1c43b717a792e2c5bbb9c93c24463d14350223) and [3](https://github.com/jonanp/ForkHub/commit/b3893d2fc509b716327257d7f6d4c218aa037ea2). Feel free to cherry-pick the commits you need.

**Comment by fadils** [comment]
@jonanp could you please tell me the steps to reproduce it?

**Comment by jonanp** [comment]
Simply add a comment in an issue. You wont see it appearing until you refresh.

**Pull request merged**